pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -375,14 +375,6 @@ func newControllerNode(cfg VSphereConfig) (*VSphere, error) {
 	if cfg.Global.VCenterPort == "" {
 		cfg.Global.VCenterPort = "443"
 	}
-	if cfg.Global.VMUUID == "" {
-		// This needs root privileges on the host, and will fail otherwise.
-		cfg.Global.VMUUID, err = getvmUUID()
```

Comment:
Shouldn't we remote getvmUUID() method as well? Which other code path is it getting used?

Reply:
True, missed this one. I have removed it in the latest commit. Please verify.